poetry.lock

```diff
@@ -1,101 +1,10 @@
-# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
this line being removed is a sign for using an outdated version of poetry. Please see [here](https://panw-global.slack.com/archives/G011E63JXPB/p1681918675380719).

Reply:
I locked locally and committed to this branch